docs/api/options.md

```diff
@@ -34,9 +34,19 @@
 - `loaders`: Object. See [defineLoaders](#appgraphqlextendschemaschema-appgraphqldefineresolversresolvers-and-appgraphqldefineloadersloaders) for more
   details.
 - `schemaTransforms`: Array of schema-transformation functions. Accept a schema as an argument and return a schema.
-- `graphiql`: boolean | string. Serve
+- `graphiql`: boolean | string | Object. Serve
```

Comment:
a question on API design here. in the same way as providing a string implies that the plugin is enable, shouldn't providing a configuration object also imply that the plugin is enabled?

Reply:
I think it's easier to manage multiple environment. With a single configuration and just a `enable` attributo to remove the plugin from the `production` env.

Reply:
Updated the opts to to be enabled by default